SConstruct

```diff
@@ -114,11 +114,16 @@ elif is_windows_host:
 		"glew32.dll",
 		"opengl32",
 	])
-else:
+elif 'steamrt_scout' in chroot_name:
 	env.Append(LIBS = [
 		"GL",
```

Comment:
I was thinking of doing it like this: `"GL" if 'steamrt_scout' in chroot_name else "OpenGL"` to avoid duplication. (And we can do the same thing for GLES above).